python-uwsgi/Dockerfile

```diff
@@ -1,18 +1,31 @@
-from unit9/base:latest
-maintainer Kamil Cholewiński <kamil.cholewinski@unit9.com>
+FROM unit9/base:latest
+
+MAINTAINER Kamil Cholewiński <kamil.cholewinski@unit9.com>
+MAINTAINER Artiom Vasiliev <artiom@unit9.com>
 
 # Dependencies
-run apt-get update && \
+RUN apt-get update && \
     apt-get install --yes --no-install-recommends \
             python-all-dev \
             python-pip \
             python-wheel \
             uwsgi-plugin-python \
+            lsb-release \
     && rm -rf /var/cache/apt
 
+RUN export CLOUD_SDK_REPO="cloud-sdk-$(lsb_release -c -s)" \
```

Comment:
Why would you *need* this here? You're bloating a single-purposed base image, that doesn't need this 99.9% of the time